.github/workflows/docker_publish.yml

```diff
@@ -5,35 +5,21 @@ on:
   workflow_dispatch:
   push:
     paths:
-      - 'CI/Dockerfile'
-      - '.github/workflows/docker_publish.yml'
+      - "CI/Dockerfile"
+      - ".github/workflows/docker_publish.yml"
 
 jobs:
   build-dependency-and-test-img:
     runs-on: ubuntu-latest
 
     strategy:
       matrix:
-        ubuntu_versions : [
-          20.04,
-          22.04,
-          ]
-        compiler : [
-          gcc,
-          clang,
-          ]
-        hdf5_versions : [
-         1.10.4,
-        ]
-        moab_versions : [
-         5.3.0,
-        ]
-        double_down : [
-          OFF,
-        ]
-        geant_version : [
-          10.7.4,
-        ]
+        ubuntu_versions: [20.04, 22.04]
+        compiler: [gcc, clang]
+        hdf5_versions: [1.10.4]
+        moab_versions: [5.3.0, 5.4.0]
+        double_down: [OFF]
+        geant_version: [10.7.4, 11.1.2]
```

Comment:
We prefer the original formatting because it makes it clearer to see where you add/remove individual entries

Reply:
Ack